kpi/urls/__init__.py

```diff
@@ -33,7 +33,7 @@
         'post': 'grant_default_model_level_perms',
     }), name='currentuser-detail'),
     re_path(r'^', include(router_api_v1.urls)),
-    re_path(r'^api/v2/', include((router_api_v2.urls, URL_NAMESPACE))),
+    re_path(r'^api/v2/', include(router_api_v2.urls)),
```

Comment:
Same comment as above
https://github.com/kobotoolbox/kpi/blob/fbc6b6b3609a22fc66f44cfe2af65b523cd705cb/kpi/urls/__init__.py#L33

Reply:
Am not sure if we decided to remove this, but it was the fix we used.
This change was because, with the namespace defined here, the reverse function failed [here](https://github.com/kobotoolbox/kpi/blob/fbc6b6b3609a22fc66f44cfe2af65b523cd705cb/kpi/serializers/v2/permission.py#L116).

Reply:
I'd suggest we use `api_v2:<view_name>` since this is the same way it's currently being done in some areas of KPI

https://github.com/kobotoolbox/kpi/blob/8b930f5d871feafd31077327b311396d7108c2c4/kpi/tests/api/v2/test_api_permissions.py#L271-L279

Reply:
Made this change.